Mathlib/Combinatorics/Enumerative/Composition.lean

```diff
@@ -145,19 +150,25 @@ theorem blocks_length : c.blocks.length = c.length :=
 functions using compositions, this is the main player. -/
 def blocksFun : Fin c.length → ℕ := c.blocks.get
 
+@[simp]
 theorem ofFn_blocksFun : ofFn c.blocksFun = c.blocks :=
   ofFn_get _
 
+@[simp]
 theorem sum_blocksFun : ∑ i, c.blocksFun i = n := by
   conv_rhs => rw [← c.blocks_sum, ← ofFn_blocksFun, sum_ofFn]
 
+@[simp]
 theorem blocksFun_mem_blocks (i : Fin c.length) : c.blocksFun i ∈ c.blocks :=
   get_mem _ _
 
-@[simp]
```

Comment:
I'm removing a `@[simp]` attribute here, because it resulted in a goal `i ∈ Composition.blocks ?c` with unknown `?c`.